src/main/java/de/fau/amos4/util/EmailSender.java

```diff
@@ -0,0 +1,56 @@
+package de.fau.amos4.util;
+import java.util.Properties;
+
+import javax.mail.*;
+import javax.mail.internet.*;
+
+public class EmailSender {
+	// Implements basic email sending functionality
+	
+	// The "from" Email address in the sent mesasges
+	// TO DO: Configuration settings for Sender, MailServerHost and Credentials
+	private String SenderEmail = "PersonalFragebogen2.0@gmail.com";	
```

Comment:
In general: It would be cool if we would stick to the naming conventions in Java. Mainly camelCase variables.